Print_A_Bot/Print_A_Bot/settings.py

```diff
@@ -64,12 +64,24 @@
 
 
 # RPi LED control
-# LED A and B are pin numbers..
-LED_A = 5
-LED_B = 10
+LED_FREQ = 100
```

Comment:
tweaked these to function with an RGB led

each color requires 1 pin vs a single color led that works from 1